lib/datapackage.py

```diff
@@ -110,18 +124,20 @@ def create_json():
                     while True:
                         # loop to check for invalid input
                         try:
-                            if val.strip() == "":  # User wants to skip
+                            if val == "":  # User wants to skip
                                 break
                             elif type(eval(val)) == int:
                                 table['dialect'][key] = val
                                 break
                             else:
+
                                 print("\nWrong type. Either leave blank or try again.\n")
                         except:
                             print("Exception occured. Try Again.\n")
                             val = input(d_opts[i] + ": ")
 
             table['schema'] = {}
+
             table['schema']["fields"] = []
             print("Enter columns [format = name, type, (optional) size]:\n")
             col = input()
```

Comment:
we could also create our own get_input (prompt, strip=True), not really important thou